### ISSUE TITLE
Visual artifacts in animated stickers

### ISSUE THREAD

**Issue by Christoph142** [UnigramDev/Unigram#1222]
Animated stickers have visual artifacts on my device (Huawei Matebook X Pro).
Happens with all sticker packs:
![sticker_artifacts](https://user-images.githubusercontent.com/1440823/67517186-4daf9c00-f6a2-11e9-8285-e7ee4bc02eb3.gif)

**Comment by entdark** [comment]
I have the same Issue on Windows 1809.

**Comment by FrayxRulez** [comment]
I’m trying hard to figure out how this happens. In the animated stickers test project everything works fine of course and debugging this kind of issues within Unigram solution is really hard. 

The interesting thing is that the glitch seems to happen on the last/first frame. Can anybody confirm this?

**Comment by Christoph142** [comment]
@FrayxRulez yes, I also noticed that it only happens when looping anew. If looping gets disabled the artifacts don't show.

**Comment by Christoph142** [comment]
Just double-checked. There actually are artifacts. But different ones.

**Comment by FrayxRulez** [comment]
Fixed in https://github.com/FrayxRulez/RLottie.UWP/commit/2ab51f3e152b6822955a45ae94fc3f5806ad1938.

Feel free to reopen this ticket if you have similar issues!
